kong/clustering/services/sync/rpc.lua

```diff
@@ -100,7 +117,7 @@ function _M:init_cp(manager)
       return nil, err
     end
 
-    if default_namespace_version == 0 or
+    if is_empty_version(default_namespace_version) or
        default_namespace_version < latest_version then
```

Comment:
Need a comment here to note that string comparison effectively does the same as number comparison

Reply:
done.